pkg/runtimeoptions/v1/api.proto

```diff
@@ -11,4 +11,7 @@ message Options {
 	// ConfigPath specifies the filesystem location of the config file
 	// used by the runtime.
 	string config_path = 2;
+	// Blob specifies an in-memory TOML blob passed from containerd's configuration section
+	// for this runtime. This will be used if config_path is not specified.
+	bytes blob = 3;
```

Comment:
- It must be string. "A TOML file must be a valid UTF-8 encoded Unicode document." - https://toml.io/en/v1.0.0
- I'd like to make this more specific. config, config_body or config_payload?

Reply:
> I'd like to make this more specific. config, config_body or config_payload?

Renamed to `config_body`.

> It must be string.

I'm not sure about string vs []byte array. Both json and TOML libraries accept bytes as input. And `optionsTree.ToTomlString` internally marshals to byte array, and then casts it to string, which seems unnecessary overhead, []byte is valid type for transferring serialized data.